fdbserver/StorageCache.actor.cpp

```diff
@@ -43,7 +43,7 @@
 // Need to look into refactoring common code out for better code readability and to avoid duplication
 
 // TODO rename wrong_shard_server error to wrong_cache_server
-inline bool canReplyWith(Error e) {
+static inline bool canReplyWith(Error e) {
```

Comment:
Shouldn't both functions return true for the same cases? Why are there 2 definitions of this?

Reply:
The implementation has some difference, `error_code_watch_cancelled` vs `error_code_cold_cache_server`. Besides that, it seems there is no reason to have two quite similar implementations.

Reply:
Generally, maybe adding `namespace` is a better choice?

Reply:
Existing code seems to favor the `static` keyword over anonymous namespaces, but I'm happy with either

Reply:
I added `error_code_watch_cancelled` recently but yes i think its better to have a single definition/namespace to avoid confusion